src/EditorFeatures/Core/IBraceMatchingService.cs

```diff
@@ -13,5 +14,8 @@ internal interface IBraceMatchingService
         Task<BraceMatchingResult?> GetMatchingBracesAsync(Document document, int position, BraceMatchingOptions options, CancellationToken cancellationToken);
     }
 
-    internal readonly record struct BraceMatchingResult(TextSpan LeftSpan, TextSpan RightSpan);
+    [DataContract]
+    internal readonly record struct BraceMatchingResult(
```

Comment:
just wondering - what is serializing this?

Reply:
ah, so i intend for the embedded stuff to run OOP when i implement it.  so that's where this will be helpeful :)